docs/guides/docs/installation/index.mdx

```diff
@@ -51,13 +51,13 @@ This will install `forc`, `forc-client`, `forc-fmt`, `forc-lsp`, `forc-wallet` a
 
 👉 Just paste the following line in your terminal and press *Enter*.
 
-{/*install_fuelup_command:example:start*/}
+{/*ANCHOR: install_fuelup_command*/}
 
 ```sh
 curl --proto '=https' --tlsv1.2 -sSf https://install.fuel.network/fuelup-init.sh | sh

```

Comment:
Please consider that "curl -sSf https://install.fuel.network/fuelup-init.sh | sh" work as expected since: 

-  The url scheme definition should be enough to enforce the HTTPS protocol 
- Curl will negotiate the highest SSL supported by the server.
- It seems that --proto '=https' --tlsv1.2 can be removed right away.

I would love to hear your thoughts on this.

Reply:
Good point for sure but I don't think this is relevant in this PR. Will address this separately :)